libcalico-go/lib/backend/k8s/conversion/workload_endpoint_default.go

```diff
@@ -34,6 +36,30 @@ import (
 	cnet "github.com/projectcalico/calico/libcalico-go/lib/net"
 )
 
+const (
+	qosk8sIngressBandwidthAnnotation   = "kubernetes.io/ingress-bandwidth"
+	qosk8sEgressBandwidthAnnotation    = "kubernetes.io/egress-bandwidth"
+	qosIngressBandwidthAnnotation      = "qos.projectcalico.org/ingressBandwidth"
```

Comment:
Please can you define these as public constants in `libcalico-go/lib/backend/k8s/conversion/constants.go`  It's useful to have them all in one place and the public constants can be useful for tests.